app/controller/membership.js

```diff
@@ -0,0 +1,246 @@
+import { wrap as coroutine } from 'co';
+import config from '../config';
+import { required, buildUrl, print, convertToObjectId } from '../components/custom-utils';
+import { insert as saveToDb, findAndUpdate } from '../components/db/service';
+import {
+    checkout as paypalCheckout,
+    accept as paypalAccept,
+    getCheckoutExperiences,
+    createCheckoutExperienceWithNoShipping
+} from '../components/paypal-helper';
+
+export const membership = (req, res) => {
```

Comment:
This page is not needed anymore because our memberships CTAs now open a modal instead of going to a page.